### PR TITLE
Update docs for supporting video attachment for notifications on Android

### PR DESCRIPTION
**Pull request by NickM-27**

### CODE REVIEW

File: docs/notifications/attachments.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Please ensure your attachment meets the criteria below, otherwise it will not sh
 | Attachment Type  | Maximum file size | Allowed Formats | Supported Platform  |
 | :-------: | --------------- | ------------------|------------------------- |
 |    Image    | 10 MB    | JPEG, GIF, PNG          | ![Android](/assets/android.svg) Android & ![iOS](/assets/iOS.svg) |
-|   Video   | 50 MB   | MPEG, MPEG2, MPEG4, AVI   | ![iOS](/assets/iOS.svg) |
+|   Video   | 50 MB   | MPEG, MPEG2, MPEG4, AVI   | ![Android](/assets/android.svg) Android & ![iOS](/assets/iOS.svg) |
```
Comment (dshokouhi) [inline docs/notifications/attachments.md:55]: We need to add a beta flag here to indicate this feature is only available in the beta.

File: docs/notifications/attachments.md [hunk 2]
```diff
@@ -67,10 +67,12 @@ You can use the following keys to add attachments. See supported media types abo
 | `image` | `/media/local/photo.jpg`<br /><br />`https://example.com/image.jpg` |
 | `audio` | `/media/local/audio.mp3`<br /><br />`https://example.com/audio.mp3` |
 
-When present, values will be used in the order of the table above. For example, you can specify a `video` and `image` and iOS will pick up the video while Android will pick up the image.
+When present, values will be used in the order of the table above. For example, you can specify an `audio` and `image` and iOS will pick up the audio while Android will pick up the image.
 
 :::info ![Android](/assets/android.svg) &nbsp; Notes:
 *   If you are setting the [`icon_url`](basic.md#notification-icon) and `image` property then only the image will be displayed on the device.
+*   If you are setting the `image`] and `video` property then only the video will be displayed on the device.
```
Comment (dshokouhi) [inline docs/notifications/attachments.md:74]: looks like a stray `]` 🙂

**Review by dshokouhi — CHANGES_REQUESTED** [review]
Just a couple minor comments, for the beta flag you can refer to a recent android PR to see how to add it.  When we cut a release we will remove the tag later.

**Comment by dshokouhi** [comment]
Also could you update the feature table at https://companion.home-assistant.io/docs/core/ we can now include check marks for both android versions!

File: docs/notifications/attachments.md (hunk 1 above)
Comment (NickM-27) [reply docs/notifications/attachments.md:55]: Do you want the beta tag to go after the android icon?

Comment (dshokouhi) [reply docs/notifications/attachments.md:55]: yea that would look better i think :)

**Review by dshokouhi — APPROVED** [review]
We don't really need the `br` tag with the beta in this spot but it doesnt hurt the design, feel free to remove that if you want

**Comment by NickM-27** [comment]
> We don't really need the `br` tag with the beta in this spot but it doesnt hurt the design, feel free to remove that if you want

Removed it 👍

**Review by JBassett — APPROVED** [review]

**Pull request merged**